packages/block-editor/README.md

```diff
@@ -798,6 +798,22 @@ _Related_
 
 -   <https://github.com/WordPress/gutenberg/blob/HEAD/packages/data/README.md#registerStore>
 
+### TextAlignControl
+
+Control to facilitate text alignment selections.
+
+_Parameters_
+
+-   _props_ `Object`: Component props.
+-   _props.className_ `string`: Class name to add to the control.
+-   _props.value_ `string`: Currently selected text align.
```

Comment:
```suggestion
-   _props.value_ `string`: Currently selected text alignment.
```
Nit: This is an example of the slightly awkward wording with "align" vs "alignment".

The line below reads better to me. Also, making the wording consistent would be a nice tweak.

Reply:
Fixed by b1dfa99 👍